pkg/kubectl/stop.go

```diff
@@ -223,6 +229,34 @@ func (reaper *DaemonSetReaper) Stop(namespace, name string, timeout time.Duratio
 	return fmt.Sprintf("%s stopped", name), nil
 }
 
+func (reaper *JobReaper) Stop(namespace, name string, timeout time.Duration, gracePeriod *api.DeleteOptions) (string, error) {
+	jobs := reaper.Experimental().Jobs(namespace)
+	scaler, err := ScalerFor("Job", NewScalerClient(*reaper))
+	if err != nil {
+		return "", err
+	}
+	job, err := jobs.Get(name)
+	if err != nil {
+		return "", err
+	}
+	if timeout == 0 {
+		// we will never have more active pods than job.Spec.Parallelism
```

Comment:
It looked like we could have more active pods if someone deleted a pod using graceful deletion.  It would be active, but ignored by the controller, which would create more.